.github/workflows/ci.yml

```
@@ -14,6 +21,10 @@ jobs:
           debug-cln
         ]
 
+        exclude:
+          - name: production-clang
+            os: macos-latest
+
```

```suggestion
            - name: debug
              os: macos-latest

```

As discussed offline, I think that it would be good to actually remove two macOS builds to speed up our CI because we only have [5 macOS workers](https://docs.github.com/en/actions/getting-started-with-github-actions/about-github-actions#usage-limits).

---

Removed all but one production build.